wiki/resources/unofficial-servers.md

```diff
@@ -62,4 +62,9 @@ __Credit:__ The Fish Project team (owned by @459235187469975572)
 ## **Dev Hub**
 > __Description:__ A community to bring developers and proficients together to share ideas and talk (Discord) meta.   <br/>
 __Link:__ (only by referral)   <br/>
-__Credit:__ Dev Hub team (owned by @386861188891279362)
\ No newline at end of file
+__Credit:__ Dev Hub team (owned by @386861188891279362)
+
+## **The Programmers Hangout** 
+> __Description:__ If you've never written code or if you have been writing code for 10 years, you're welcome here. The Programmer's Hangout (TPH) Welcomes all levels and is a great place to get a solid footing in programming.   <br/>
```

Comment:
This shouldn't be an "advertisement" with a call-to-action.
You might be familiar with[Wikipedia's writing style](https://en.wikipedia.org/wiki/Wikipedia:Writing_better_articles#Information_style_and_tone). We also prioritize information and neutrality. Remember that DRW is a curated assortment of resources, not a bot or server list.

Reply:
Before I update the branch, what do you think of the following? 

```The Programmers Hangout is a programming community with an emphasis on hangout. It caters for all levels of experience, whether you've never written code or if you have been writing code for 10 years. With a strong link to open source, education and career progression```

Reply:
I feel like that still doesn't sound like an impartial subscription.

> Social, beginner-friendly programming community prioritizing open source, education and career progression.
Would be an example.

Reply:
Ah, fair enough. We weren't sure about using you/your but I noticed displace had and thought it would be fine. 

```The Programmers Hangout is a general-purpose programming community with an emphasis on hangout, catering for all experience levels. Has a strong link to open source products, education, learning and career progression```
how about this?

Reply:
s/catering for/catering to
s/Has a strong link/<maybe something nicer to read :)>

rest lgtm